app/controllers/urls_controller.rb

```diff
@@ -5,28 +5,20 @@ def new
   end
 
   def create
-    @url = Url.new(url_params)
-    @url.clean_url
+    @url = CreateShortUrlService.new(url_params).call
```

Comment:
It's a matter of preference but I'd consider the usage of `Callable` module / class so that this line could be rewritten like `CreateShortUrlService.call(url_params)`